clients/flash/air-client/src/Default.css

```diff
@@ -103,8 +103,10 @@ main|LoadingScreen {
 
 main|MainView {
 	backgroundColor : PropertyReference("bbbBlack");
+	backgroundAlpha : 0
```

Comment:
It would be cleaner if this was a class instead and then you could swap the styleNames in MainView.onScreenshareRunning().